modules/compliance-operator-rosa-installation.adoc

```diff
@@ -0,0 +1,117 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/co-management/compliance-operator-installation.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-compliance-operator-cli_{context}"]
+= Installing the Compliance Operator on ROSA
+
+As of the Compliance Operator 1.5.0 release, the operator is tested against Red Hat
+OpenShift Service on AWS (ROSA) using hosted control plane topologies (HCP).
+
+ROSA HCP clusters restrict access to the control plane, which is managed by Red
```

Comment:
Maybe you want to say
`ROSA HCP clusters **have** restrict access to the control plane, which is managed by Red Hat`